src/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  font-family: "ROBOTO", Helvetica, sans-serif;
+}
+
+.card__img {
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+}
+
+.card__article-information {
+  margin-top: 40px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.card__name {
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  font-weight: 500;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__code--review {
```

Comment:
Remove the unnecessary '.card__code--review' class, as it is not mentioned in the task description and seems to be unused.